tests/Feature/Http/Pages/ShowPageTest.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+use App\Models\Page;
+use App\Models\User;
+use Tests\TestCase;
+
+test('404 if page not found', function () {
+    /** @var TestCase $this */
+    $response = $this->get('/pages/some-fake-slug');
+    $response->assertNotFound();
+});
+
+test('404 if page not published for non-admin', function () {
```

Comment:
This test is acting as a guest. It should act as an authenticted, non-admin user